components/brave_wallet/browser/pref_names.cc

```diff
@@ -16,6 +16,8 @@ const char kBraveWalletCurrentChainId[] =
     "brave.wallet.wallet_current_chain_id";
 const char kBraveWalletKeyrings[] = "brave.wallet.keyrings";
 const char kBraveWalletUserAssets[] = "brave.wallet.user_assets";
+const char kBraveWalletUserAssetEthContractAddressMigrated[] =
+    "brave.wallet.user.asset.eth.contract.address.migrated";
```

Comment:
brave.wallet.user_asset.eth_contract_address_migrated

Reply:
Fixed in https://github.com/brave/brave-core/pull/10333/commits/75a3d3253f5b0e0d65f6e163616dce34f5c69756